website_blog_group_old_archives/controllers/website_blog.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+from openerp import http, tools
+from openerp.http import request
+from openerp.addons.website_blog.controllers.main import WebsiteBlog
+
+class website_blog(WebsiteBlog):
+
+
+    def nav_list_grouped(self):
+        blog_post_obj = request.registry['blog.post']
```

Comment:
no need to change this now, but the `request` object has a perfectly good property called `env`